tests/plugins/test_tvplayer.py

```diff
@@ -33,20 +33,18 @@ def test_can_handle_url(self):
     @patch('streamlink.plugins.tvplayer.HLSStream')
     def test_get_streams(self, hlsstream, mock_get_stream_data):
         mock_get_stream_data.return_value = {
-            "stream": "http://test.se/stream1"
+            "response": {"stream": "http://test.se/stream1", "drm": None }
         }
 
         page_resp = Mock()
         page_resp.text = u"""
-                    <div class="video-js theoplayer-skin theo-seekbar-above-controls content-box vjs-fluid"
-                 data-resource= "bbcone"
-                 data-token = "1324567894561268987948596154656418448489159"
-                                    data-content-type="live"
-                    data-environment="live"
-                    data-subscription="free"
-                    data-channel-id="89">
-                <div id="channel-info" class="channel-info">
-                    <div class="row visible-xs visible-sm">
+                    data-player-library="videojs"
+                                            data-player-id="ILWmaLKV41Ql8kF"
+                                        data-player-key="2Pw1Eg0Px3Dy9Jm3Ly8Dr5Bi5Vc5Nk"
+                    data-player-uvid="139"
+                    data-player-token="294c808a378a09d6e36d0253ab3765af"
+                    data-player-expiry="1581958189"
+                    data-player-poster=""
```

Comment:
Please fix the indentation. Also add the wrapping HTMLElement tag and not just its attributes, even though it's not required by the plugin's regex. Otherwise, reading the test can be confusing.

Reply:
OK.  One question - should I apply PEP8 on the long lines in the HTML or leave it as is?

Reply:
Also, I assume indentation here should be:
```
    thing = u"""
        <div>
            blah
        </div>
    """
```

And not:

```
    thing = u"""
<div>
    blah
</div>
    """
```